test/tests/functional/pbs_mom_walltime.py

```diff
@@ -147,17 +147,26 @@ def test_suspend_time_not_counted_in_walltime(self):
         self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
                             {'scheduling': 'True'})
 
-        time.sleep(15)
+        time.sleep(30)
```

Comment:
please add a comment as why are you sleeping for 30s.

Reply:
I always am fond of logging why you are sleeping instead of adding a comment.  The same text is in the code, but it gives the person running a test a reason why their test stalled for 30s.